### PR TITLE
Add the html-css for Survey form

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
Hello there...
Here are the changes made to this repository
- I created a repo for the paired programming with my previous partner
- I copied the codes and committed
- I added linters to check errors
- I pushed all files to the repository
Anticipating a positive review
Thanks and happy reviewing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 id="title"> freeCodeCamp Survey form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+      <div class="div-form">
+        <label for="name" id="name-label">Name</label>
+        <input type="text" id="name" placeholder="Enter the name" required />
+      </div>
```
Comment (PromzzyKoncepts) [inline index.html:17]: Please also add a descriptive value for action, the attribute should not be empty. This makes your code unclean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          id="number"
+          min="18"
+          max="100"
+          placeholder="Enter your age"
+          required
+        />
+      </div>
+      <div class="div-form">
+        <label for="role" id="role-label"
+          >Which option best describes your current role?</label>
```
Comment (PromzzyKoncepts) [inline index.html:40]: # Changes Required ♻️ :
Please make changes to the value of the attribute "for", the value of the “for” attribute of the “label” element should be the ID of a non-hidden form control. This is a more professional way of writing a label tag.
Thanks 😁

File: index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        <select name="role" id="dropdown">
+          <option value="empty"></option>
+          <option value="select here" disabled>Select here</option>
+          <option value="Student">Student</option>
```
Comment (PromzzyKoncepts) [inline index.html:44]: Please make changes to the option tag, element “option” without attribute “label” must not be empty.
Thank you 😀

**Review by PromzzyKoncepts — COMMENTED** [review]
# Great Work 👍 
<h2>You are almost there</h2>

Here are some good reviews first
- Good commit messages.
- Great Github workflow.
- Awesome finished results.


# Changes Required ♻️ 

- Linters did not run completely.
- Make changes in your HTML documents according to the linter errors.
- Check below for the changes I have stated in your code.

- Don't forget to tag me in your code if you have any questions concerning the reviews made.
- You can submit for a re-review after the changes have been made.

<h2>Happy Coding</h2>

File: index.html (hunk 2 above)
Comment (Kossi-stack) [reply index.html:40]: - [ ] Kindly remove changes required here

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @PromzzyKoncepts,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] Kindly edit the PR's title to something more descriptive. it can be "Review Past Project PR".

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**